Cargo.lock

```diff
@@ -1443,7 +1443,7 @@ dependencies = [
  "socket2",
  "widestring",
  "windows-sys 0.48.0",
- "winreg 0.50.0",
+ "winreg",
```

Comment:
Unfortunatelly, we still have `winreg` in our dependency tree because of the `ipconfig` crate. it hasn't updated in two years (https://github.com/liranringel/ipconfig) and used by the DNS resolver we use (`hickory-resolver`)

Reply:
It is what it is…! Maybe one day this will be removed from the dependency tree. At least, we only have one winreg instead of two :joy: